### ISSUE TITLE
Add classifier cross-validation script

### ISSUE THREAD

**Issue by colettace** [ome/omero-wndcharm#18]
This is definitely not a lot of work. All that is required is a new OMERO.WND-CHARM wrapper script called $OMERO_PYCHRM_HOME/scripts/Pychrm_classifier_cross_validate.py. Copy/paste the code from $WNDCHARM_HOME/examples/classifier_cross_validation.py in there. Provide text boxes for user to input number of cross validation splits, training set/test set split %, etc. These user inputs are fed directly into Pychrm calls without further processing. The output of DiscreteClassificationExperimentResult.Print() can go into a text file annotation on the project.

**Comment by manics** [comment]
In https://github.com/manics/omero-pychrm/pull/30 running the cross-validation script results in an error after printing out the features:

```
===========================================
Experiment name: None
Summary:
Number of batches: 5
Total number of classifications: 30
Total number of CORRECT classifications: 25
Total classification accuracy: 0.8333

Batch Accuracies:
#   Acc.    Name
------------------------------------
2   1.0000  None
3   1.0000  None
0   0.8333  None
4   0.8333  None
1   0.5000  None
Feature Weight Analysis:
Rank    mean    count   StdDev  Min Max Name
----    ----    -----   ------  --- --- ----
1   220.070   5 432.065 3.948   1084.200    [0] Comb Moments (Wavelet (Fourier ())) [16]
2   220.070   5 432.065 3.948   1084.200    [0] Comb Moments (Wavelet (Fourier ())) [17]
3   28.812    5 26.672  6.589   70.428  [0] Multiscale Histograms (Chebyshev (Fourier ())) [23]
4   28.748    5 26.725  6.362   70.428  [0] Multiscale Histograms (Chebyshev (Fourier ())) [14]
5   18.201    5 4.352   15.334  26.834  [0] Zernike Coefficients (Fourier ()) [3]
6   17.979    5 5.869   13.978  29.656  [0] Zernike Coefficients (Fourier ()) [33]
7   17.468    5 4.540   14.079  26.304  [0] Zernike Coefficients (Fourier ()) [7]
8   17.411    5 4.374   12.127  25.207  [0] Haralick Textures (Chebyshev (Fourier ())) [24]
9   17.254    5 3.800   14.400  24.344  [0] Zernike Coefficients (Fourier ()) [45]
10  16.540    5 6.323   11.342  28.673  [0] Zernike Coefficients (Fourier ()) [23]
11  16.462    5 4.043   9.950   22.073  [0] Haralick Textures (Wavelet ()) [12]
12  16.331    5 3.320   13.002  22.050  [0] Zernike Coefficients (Fourier ()) [31]
13  15.490    5 2.621   11.228  18.387  [0] Haralick Textures () [14]
14  15.000    5 1.176   13.103  16.735  [0] Haralick Textures () [8]
15  14.953    5 2.993   11.635  20.302  [0] Zernike Coefficients (Fourier ()) [13]
16  14.606    5 7.272   7.992   27.243  [0] Zernike Coefficients (Fourier ()) [53]
17  14.229    5 2.529   10.259  17.766  [0] Haralick Textures () [10]
18  13.840    5 3.285   10.148  19.980  [0] Zernike Coefficients (Fourier ()) [57]
19  13.599    5 4.290   8.631   19.697  [0] Haralick Textures (Chebyshev (Fourier ())) [6]
20  13.093    5 5.279   7.640   22.465  [0] Zernike Coefficients (Fourier ()) [40]
21  13.060    5 5.588   8.325   22.161  [0] Zernike Coefficients (Fourier ()) [61]
22  13.054    5 2.529   10.983  18.030  [0] Haralick Textures (Wavelet ()) [8]
23  12.973    5 2.290   9.126   16.015  [0] Zernike Coefficients (Fourier ()) [44]
24  12.817    5 1.156   11.719  14.258  [0] Haralick Textures () [12]
25  12.817    5 2.961   10.445  18.172  [0] Haralick Textures (Chebyshev (Fourier ())) [20]
26  12.573    5 2.723   8.657   16.094  [0] Zernike Coefficients (Fourier ()) [32]
27  12.215    5 3.789   8.693   18.103  [0] Tamura Textures () [2]
28  12.174    5 4.410   6.415   17.770  [0] Tamura Textures (Wavelet (Fourier ())) [0]
29  12.153    5 3.889   6.388   16.927  [0] Tamura Textures (Wavelet (Fourier ())) [5]
30  12.093    5 1.960   8.249   13.562  [0] Tamura Textures (Wavelet ()) [1]
31  11.914    5 4.528   6.682   17.710  [0] Haralick Textures (Chebyshev (Fourier ())) [2]
32  11.904    5 1.046   10.274  13.477  [0] Zernike Coefficients (Fourier ()) [43]
33  11.855    4 5.084   7.246   20.393  [0] Chebyshev Coefficients (Fourier ()) [11]
34  11.837    5 2.406   9.191   16.272  [0] Haralick Textures (Wavelet ()) [10]
35  11.487    5 4.671   6.018   17.254  [0] Haralick Textures (Chebyshev (Fourier ())) [26]
36  11.357    5 3.137   7.161   15.287  [0] Zernike Coefficients (Fourier ()) [70]
37  11.213    5 3.494   7.970   18.016  [0] Zernike Coefficients (Fourier ()) [62]
38  11.101    5 2.953   7.466   15.645  [0] Zernike Coefficients (Fourier ()) [68]
39  10.799    5 3.685   7.785   18.055  [0] Zernike Coefficients (Fourier ()) [34]
40  10.638    5 6.051   3.191   20.617  [0] Multiscale Histograms (Chebyshev (Fourier ())) [13]
41  10.632    5 3.443   8.083   17.447  [0] Zernike Coefficients (Fourier ()) [8]
42  10.624    5 3.517   7.840   17.551  [0] Zernike Coefficients (Fourier ()) [24]
43  10.619    5 3.196   7.843   16.817  [0] Zernike Coefficients (Fourier ()) [48]
44  10.582    5 2.874   6.842   14.966  [0] Zernike Coefficients (Fourier ()) [29]
45  10.440    5 4.494   5.350   16.434  [0] Haralick Textures (Chebyshev (Fourier ())) [7]
46  10.391    5 2.483   8.147   15.010  [0] Zernike Coefficients (Fourier ()) [2]
47  10.384    5 13.180  3.696   36.744  [0] Haralick Textures (Wavelet ()) [4]
48  10.378    5 2.503   8.127   14.997  [0] Zernike Coefficients (Fourier ()) [0]
49  10.292    5 3.510   7.646   17.179  [0] Zernike Coefficients (Fourier ()) [14]
50  10.218    5 1.851   7.031   12.312  [0] Haralick Textures (Wavelet ()) [14]
51  10.084    5 2.630   6.959   14.599  [0] Zernike Coefficients (Fourier ()) [59]
52  10.083    5 3.701   7.007   17.222  [0] Zernike Coefficients (Fourier ()) [46]
53  10.054    5 5.166   4.056   15.977  [0] Haralick Textures (Chebyshev (Fourier ())) [3]
54  10.000    5 2.715   6.843   15.000  [0] Zernike Coefficients (Fourier ()) [58]
55  9.902     5 2.031   7.789   13.278  [0] Zernike Coefficients (Fourier ()) [6]
56  9.867     5 1.800   7.960   12.517  [0] Tamura Textures () [5]
57  9.431     4 4.774   5.100   18.366  [0] Multiscale Histograms (Chebyshev (Fourier ())) [18]
58  9.384     5 4.487   4.376   14.603  [0] Tamura Textures (Chebyshev (Fourier ())) [3]
59  9.335     5 2.606   6.184   13.667  [0] Zernike Coefficients (Fourier ()) [55]
60  9.050     5 2.253   6.778   13.168  [0] Haralick Textures (Wavelet ()) [22]
61  8.791     5 1.684   6.573   10.747  [0] Zernike Coefficients (Fourier ()) [20]
62  8.766     5 2.561   5.503   11.898  [0] Zernike Coefficients (Fourier ()) [15]
63  8.523     5 4.006   5.616   16.430  [0] Zernike Coefficients (Fourier ()) [19]
64  8.490     5 1.822   6.163   11.427  [0] Edge Features () [0]
65  8.471     5 2.105   6.322   12.258  [0] Zernike Coefficients (Fourier ()) [5]
66  8.438     5 1.490   6.278   10.410  [0] Haralick Textures (Wavelet ()) [9]
67  8.429     5 1.850   5.658   10.781  [0] Zernike Coefficients (Fourier ()) [56]
68  8.380     5 2.997   5.985   13.591  [0] Tamura Textures (Chebyshev ()) [5]
69  8.256     5 1.878   5.738   10.544  [0] Haralick Textures (Wavelet ()) [13]
70  8.193     5 1.791   6.434   11.411  [0] Zernike Coefficients (Fourier ()) [12]
71  7.833     4 4.930   4.541   16.325  [0] Multiscale Histograms (Chebyshev (Fourier ())) [10]
72  7.726     5 2.289   5.823   11.115  [0] Tamura Textures (Chebyshev ()) [0]
73  7.666     5 2.676   4.452   11.695  [0] Chebyshev Coefficients (Fourier ()) [12]
74  7.547     5 1.445   5.990   9.959   [0] Zernike Coefficients (Fourier ()) [30]
75  7.508     5 3.561   5.374   14.618  [0] Tamura Textures () [1]
76  7.365     5 3.010   5.086   13.313  [0] Haralick Textures (Chebyshev (Fourier ())) [10]
77  7.217     5 1.627   5.657   9.661   [0] Edge Features () [18]
78  7.067     5 1.782   4.578   9.809   [0] Haralick Textures (Wavelet ()) [7]
79  7.011     5 2.197   4.136   9.878   [0] Tamura Textures (Chebyshev (Fourier ())) [4]
80  7.001     5 3.531   4.738   14.019  [0] Zernike Coefficients (Fourier ()) [41]
81  6.994     5 2.365   4.153   9.973   [0] Zernike Coefficients (Fourier ()) [21]
82  6.850     5 0.854   5.909   8.429   [0] Tamura Textures (Wavelet ()) [5]
83  6.835     5 2.040   4.090   9.601   [0] Zernike Coefficients (Fourier ()) [11]
84  6.811     5 2.335   4.492   11.133  [0] Haralick Textures (Wavelet ()) [26]
85  6.772     5 3.515   4.294   13.704  [0] Zernike Coefficients (Fourier ()) [28]
86  6.698     5 1.214   5.320   8.450   [0] Comb Moments (Wavelet (Fourier ())) [39]
87  6.636     4 2.704   4.322   11.746  [0] Multiscale Histograms (Chebyshev (Fourier ())) [1]
88  6.617     5 1.191   5.158   8.387   [0] Zernike Coefficients (Fourier ()) [10]
89  6.427     5 1.868   4.607   9.779   [0] Comb Moments (Fourier ()) [6]
90  6.408     3 1.839   8.102   12.263  [0] Haralick Textures (Chebyshev (Fourier ())) [25]
91  6.379     5 1.249   4.927   8.127   [0] Haralick Textures (Wavelet ()) [18]
92  6.185     5 2.967   4.073   12.058  [0] Haralick Textures (Chebyshev (Fourier ())) [22]
93  6.151     5 3.314   3.433   12.586  [0] Zernike Coefficients (Fourier ()) [71]
94  6.123     5 1.647   4.702   9.102   [0] Edge Features () [26]
95  6.099     5 2.886   3.707   11.752  [0] Haralick Textures (Wavelet ()) [27]
96  5.827     5 1.088   3.772   6.656   [0] Haralick Textures (Wavelet ()) [21]
97  5.782     5 2.446   3.752   10.545  [0] Haralick Textures (Chebyshev (Fourier ())) [8]
98  5.763     5 2.624   3.572   10.813  [0] Zernike Coefficients (Fourier ()) [37]
99  5.617     5 0.862   4.616   7.139   [0] Tamura Textures (Wavelet ()) [2]
100 5.366     5 1.219   4.072   7.621   [0] Edge Features () [17]
101 5.363     5 1.741   3.617   8.618   [0] Comb Moments (Chebyshev ()) [12]
102 5.291     5 1.040   3.790   6.805   [0] Zernike Coefficients () [66]
103 5.273     5 1.421   3.930   7.833   [0] Edge Features () [25]
104 5.219     5 1.009   4.027   6.696   [0] Edge Features () [19]
105 5.195     5 2.497   2.815   9.465   [0] Multiscale Histograms (Chebyshev (Fourier ())) [7]
106 5.105     4 2.363   3.338   9.969   [0] Comb Moments (Chebyshev ()) [2]
107 4.843     4 2.875   3.030   10.569  [0] Multiscale Histograms (Chebyshev (Fourier ())) [2]
108 4.771     4 2.827   3.105   10.479  [0] Multiscale Histograms (Chebyshev (Fourier ())) [21]
109 4.665     4 1.126   4.446   7.587   [0] Haralick Textures (Wavelet ()) [3]
110 4.658     4 1.228   3.787   7.017   [0] Haralick Textures (Wavelet ()) [19]
111 4.654     4 2.174   3.498   8.037   [0] Zernike Coefficients (Fourier ()) [67]
112 4.562     4 0.928   4.822   7.110   [0] Zernike Coefficients (Fourier ()) [18]
113 4.442     5 1.104   3.441   6.473   [0] Haralick Textures () [6]
114 4.367     4 1.899   3.457   8.378   [0] Multiscale Histograms (Chebyshev (Fourier ())) [0]
115 4.358     4 0.698   4.507   6.131   [0] Tamura Textures (Wavelet ()) [3]
116 4.333     5 1.073   3.523   6.453   [0] Haralick Textures (Wavelet ()) [11]
117 4.332     5 0.681   3.098   5.039   [0] Zernike Coefficients (Fourier ()) [27]
118 4.234     3 2.056   4.246   9.105   [0] Multiscale Histograms (Chebyshev (Fourier ())) [9]
119 4.108     4 0.826   3.828   6.042   [0] Zernike Coefficients (Fourier ()) [42]
120 4.051     5 0.375   3.464   4.547   [0] Tamura Textures (Fourier ()) [5]
121 3.986     3 1.734   4.202   8.061   [0] Multiscale Histograms (Chebyshev (Fourier ())) [5]
122 3.809     3 2.149   3.394   8.447   [0] Multiscale Histograms (Wavelet ()) [7]
123 3.765     4 1.044   3.846   6.488   [0] Haralick Textures () [24]
124 3.675     4 2.362   2.804   8.635   [0] Radon Coefficients (Fourier ()) [3]
125 3.632     3 1.574   4.536   8.223   [0] Multiscale Histograms (Wavelet ()) [13]
126 3.484     2 2.397   6.313   11.108  [0] Multiscale Histograms (Wavelet ()) [22]
127 3.472     4 1.071   2.840   5.534   [0] Multiscale Histograms (Chebyshev (Fourier ())) [6]
128 3.423     3 2.713   3.285   9.493   [0] Zernike Coefficients (Fourier ()) [39]
129 3.240     1 0.000   16.200  16.200  [0] Comb Moments (Wavelet (Fourier ())) [11]
130 3.204     3 2.172   3.486   8.388   [0] Zernike Coefficients (Fourier ()) [54]
131 3.117     3 1.950   3.386   7.903   [0] Haralick Textures () [20]
132 3.077     3 2.087   3.620   8.080   [0] Multiscale Histograms () [4]
133 2.988     3 0.813   3.987   5.978   [0] Haralick Textures () [2]
134 2.899     3 1.866   3.513   7.471   [0] Haralick Textures (Chebyshev (Fourier ())) [0]
135 2.860     3 1.310   3.602   6.596   [0] Multiscale Histograms () [17]
136 2.713     3 0.476   3.855   4.935   [0] Zernike Coefficients (Fourier ()) [47]
137 2.702     3 0.352   4.180   4.992   [0] Haralick Textures () [11]
138 2.698     3 0.195   4.272   4.748   [0] Zernike Coefficients (Fourier ()) [64]
139 2.646     1 0.000   13.229  13.229  [0] Comb Moments (Wavelet (Fourier ())) [21]
140 2.596     3 0.461   3.723   4.842   [0] Haralick Textures (Wavelet ()) [0]
141 2.592     3 1.111   3.132   5.804   [0] Edge Features () [11]
142 2.518     3 0.686   3.638   5.163   [0] Edge Features () [4]
143 2.502     3 0.267   3.892   4.531   [0] Haralick Textures () [13]
144 2.450     3 0.949   3.124   5.376   [0] Comb Moments (Fourier ()) [7]
145 2.432     3 0.214   3.754   4.245   [0] Multiscale Histograms (Wavelet ()) [2]
146 2.373     3 0.549   3.235   4.566   [0] Comb Moments (Chebyshev ()) [0]
147 2.325     2 2.322   3.489   8.133   [0] Radon Coefficients (Fourier ()) [4]
148 2.308     3 0.423   3.412   4.420   [0] Chebyshev Coefficients (Fourier ()) [3]
149 2.304     2 1.560   4.200   7.320   [0] Radon Coefficients (Fourier ()) [5]
150 2.271     2 1.047   4.629   6.724   [0] Chebyshev Coefficients (Fourier ()) [2]
151 2.204     3 0.160   3.512   3.892   [0] Tamura Textures (Fourier ()) [2]
152 2.152     2 0.559   4.822   5.940   [0] Haralick Textures (Chebyshev (Fourier ())) [27]
153 2.129     2 0.062   5.261   5.385   [0] Multiscale Histograms (Chebyshev (Fourier ())) [11]
154 2.117     3 0.506   3.062   4.232   [0] Comb Moments (Chebyshev ()) [14]
155 2.086     2 0.100   5.114   5.315   [0] Chebyshev Coefficients (Fourier ()) [14]
156 2.042     2 0.504   4.601   5.609   [0] Multiscale Histograms () [10]
157 2.021     2 1.915   3.138   6.967   [0] Zernike Coefficients (Fourier ()) [38]
158 1.947     2 1.667   3.200   6.533   [0] Comb Moments (Fourier ()) [16]
159 1.947     2 1.667   3.200   6.533   [0] Comb Moments (Fourier ()) [15]
160 1.860     2 1.437   3.212   6.086   [0] Chebyshev Coefficients (Fourier ()) [1]
161 1.788     2 0.258   4.212   4.729   [0] Zernike Coefficients (Fourier ()) [49]
162 1.748     2 0.072   4.297   4.441   [0] Multiscale Histograms (Chebyshev (Fourier ())) [19]
163 1.745     2 0.999   3.363   5.362   [0] Haralick Textures () [4]
164 1.720     2 0.210   4.090   4.509   [0] Zernike Coefficients (Fourier ()) [50]
165 1.707     1 0.000   8.533   8.533   [0] Comb Moments (Fourier ()) [3]
166 1.707     1 0.000   8.533   8.533   [0] Comb Moments (Fourier ()) [4]
167 1.640     2 0.448   3.653   4.548   [0] Chebyshev Coefficients (Fourier ()) [15]
168 1.603     2 0.062   3.945   4.069   [0] Multiscale Histograms (Chebyshev (Fourier ())) [22]
169 1.575     2 0.077   3.861   4.015   [0] Chebyshev Coefficients (Fourier ()) [13]
170 1.564     2 0.082   3.828   3.991   [0] Edge Features () [8]
171 1.561     2 0.344   3.558   4.246   [0] Multiscale Histograms () [18]
172 1.544     2 0.633   3.226   4.493   [0] Haralick Textures () [3]
173 1.447     2 0.261   3.357   3.880   [0] Haralick Textures (Wavelet ()) [2]
174 1.435     2 0.125   3.464   3.713   [0] Comb Moments (Chebyshev ()) [25]
175 1.433     2 0.507   3.075   4.089   [0] Otsu Object Features () [0]
176 1.432     2 0.505   3.074   4.084   [0] Otsu Object Features () [17]
177 1.428     2 0.305   3.264   3.874   [0] Edge Features () [20]
178 1.422     1 0.000   7.111   7.111   [0] Multiscale Histograms () [1]
179 1.416     2 0.076   3.463   3.616   [0] Multiscale Histograms (Chebyshev (Fourier ())) [12]
180 1.410     2 0.022   3.503   3.547   [0] Comb Moments (Wavelet ()) [12]
181 1.385     2 0.245   3.219   3.709   [0] Chebyshev Coefficients (Fourier ()) [10]
182 1.375     1 0.000   6.873   6.873   [0] Comb Moments (Wavelet (Fourier ())) [22]
183 1.359     1 0.000   6.794   6.794   [0] Haralick Textures (Chebyshev (Fourier ())) [14]
184 1.355     2 0.145   3.243   3.533   [0] Haralick Textures (Wavelet ()) [6]
185 1.326     2 0.371   2.943   3.685   [0] Haralick Textures () [22]
186 1.296     1 0.000   6.480   6.480   [0] Edge Features () [7]
187 1.266     1 0.000   6.328   6.328   [0] Inverse-Otsu Object Features () [0]
188 1.253     1 0.000   6.263   6.263   [0] Inverse-Otsu Object Features () [17]
189 1.189     1 0.000   5.945   5.945   [0] Comb Moments (Wavelet ()) [37]
190 1.182     1 0.000   5.911   5.911   [0] Radon Coefficients (Fourier ()) [9]
191 1.127     1 0.000   5.634   5.634   [0] Haralick Textures (Fourier ()) [0]
192 1.072     1 0.000   5.360   5.360   [0] Comb Moments (Wavelet (Fourier ())) [23]
193 1.058     1 0.000   5.291   5.291   [0] Radon Coefficients (Fourier ()) [10]
194 1.054     1 0.000   5.268   5.268   [0] Zernike Coefficients (Fourier ()) [65]
195 1.044     1 0.000   5.220   5.220   [0] Haralick Textures () [5]
196 0.985     1 0.000   4.923   4.923   [0] Zernike Coefficients (Fourier ()) [52]
197 0.959     1 0.000   4.794   4.794   [0] Haralick Textures (Chebyshev (Fourier ())) [1]
198 0.947     1 0.000   4.736   4.736   [0] Gabor Textures () [2]
199 0.923     1 0.000   4.615   4.615   [0] Haralick Textures (Fourier ()) [14]
200 0.919     1 0.000   4.594   4.594   [0] Tamura Textures () [4]
201 0.899     1 0.000   4.494   4.494   [0] Inverse-Otsu Object Features () [22]
202 0.891     1 0.000   4.457   4.457   [0] Haralick Textures (Fourier ()) [10]
203 0.889     1 0.000   4.443   4.443   [0] Haralick Textures (Wavelet (Fourier ())) [18]
204 0.884     1 0.000   4.421   4.421   [0] Multiscale Histograms () [9]
205 0.882     1 0.000   4.409   4.409   [0] Zernike Coefficients (Fourier ()) [69]
206 0.879     1 0.000   4.397   4.397   [0] Haralick Textures (Wavelet ()) [15]
207 0.859     1 0.000   4.297   4.297   [0] Haralick Textures (Wavelet ()) [1]
208 0.851     1 0.000   4.254   4.254   [0] Haralick Textures (Fourier ()) [22]
209 0.847     1 0.000   4.237   4.237   [0] Haralick Textures (Fourier ()) [26]
210 0.842     1 0.000   4.210   4.210   [0] Zernike Coefficients () [4]
211 0.841     1 0.000   4.206   4.206   [0] Haralick Textures (Fourier ()) [8]
212 0.840     1 0.000   4.200   4.200   [0] Comb Moments (Wavelet (Fourier ())) [29]
213 0.818     1 0.000   4.092   4.092   [0] Haralick Textures (Fourier ()) [24]
214 0.813     1 0.000   4.064   4.064   [0] Haralick Textures (Chebyshev (Fourier ())) [21]
215 0.813     1 0.000   4.063   4.063   [0] Comb Moments (Wavelet (Fourier ())) [4]
216 0.813     1 0.000   4.063   4.063   [0] Comb Moments (Wavelet (Fourier ())) [5]
217 0.806     1 0.000   4.031   4.031   [0] Inverse-Otsu Object Features () [25]
218 0.799     1 0.000   3.995   3.995   [0] Multiscale Histograms () [5]
219 0.799     1 0.000   3.994   3.994   [0] Inverse-Otsu Object Features () [23]
220 0.793     1 0.000   3.964   3.964   [0] Edge Features () [10]
221 0.786     1 0.000   3.931   3.931   [0] Haralick Textures (Chebyshev ()) [10]
222 0.785     1 0.000   3.924   3.924   [0] Zernike Coefficients (Fourier ()) [26]
223 0.781     1 0.000   3.905   3.905   [0] Edge Features () [9]
224 0.778     1 0.000   3.892   3.892   [0] Radon Coefficients (Fourier ()) [6]
225 0.775     1 0.000   3.877   3.877   [0] Multiscale Histograms (Wavelet ()) [1]
226 0.765     1 0.000   3.826   3.826   [0] Haralick Textures (Fourier ()) [6]
227 0.765     1 0.000   3.826   3.826   [0] Tamura Textures (Wavelet (Fourier ())) [1]
228 0.750     1 0.000   3.748   3.748   [0] Zernike Coefficients () [1]
229 0.737     1 0.000   3.685   3.685   [0] Haralick Textures () [7]
230 0.733     1 0.000   3.664   3.664   [0] Otsu Object Features () [11]
231 0.729     1 0.000   3.645   3.645   [0] Chebyshev Coefficients (Fourier ()) [4]
232 0.722     1 0.000   3.611   3.611   [0] Haralick Textures () [1]
233 0.712     1 0.000   3.562   3.562   [0] Comb Moments () [20]
234 0.708     1 0.000   3.539   3.539   [0] Multiscale Histograms () [7]
235 0.695     1 0.000   3.475   3.475   [0] Chebyshev Coefficients (Fourier ()) [16]
236 0.686     1 0.000   3.429   3.429   [0] Haralick Textures () [18]
237 0.658     1 0.000   3.291   3.291   [0] Haralick Textures () [0]
238 0.653     1 0.000   3.266   3.266   [0] Haralick Textures () [15]
239 0.649     1 0.000   3.246   3.246   [0] Otsu Object Features () [14]
240 0.640     1 0.000   3.200   3.200   [0] Comb Moments (Chebyshev ()) [27]
241 0.640     1 0.000   3.200   3.200   [0] Comb Moments (Chebyshev ()) [28]
242 0.640     1 0.000   3.200   3.200   [0] Radon Coefficients (Fourier ()) [8]
243 0.631     1 0.000   3.154   3.154   [0] Haralick Textures (Chebyshev ()) [13]
244 0.612     1 0.000   3.059   3.059   [0] Multiscale Histograms () [16]
245 0.602     1 0.000   3.009   3.009   [0] Zernike Coefficients () [9]
246 0.565     1 0.000   2.826   2.826   [0] Zernike Coefficients (Fourier ()) [60]

Error in experiment.PredictedValueAnalysis: unsupported operand type(s) for /: 'NoneType' and 'float'
```

**Comment by manics** [comment]
Fixed by #30
